configure

```diff
@@ -3182,8 +3182,13 @@ update_module_mk() {
     local math_cmd="${4:-math}"
 
     [ -e "$model_file" ] || return
-    if [ ! "${mkfile}" -nt "${model_file}" ] && \
-       [ ! "${mkfile}" -ot "${model_file}" ]; then
+
+    local mk_date model_date date_tolerance
+    mk_date="$(date -r "${mkfile}" +%s)"
```

Comment:
I've got a stylistic wish: In this case one could leave out the first line and instead write
~~~.bash
local mk_date="$(date -r "${mkfile}" +%s)"
local model_date="$(date -r "${model_file}" +%s)"
local date_tolerance="5" # Seconds
~~~
This is shorter by one line and in my opinion more readable, because the word `local` is directly in front of the variables. (This would also apply to the for loop in the patch below.)

Reply:
@Expander, this is error prone, because the definition and initialization of variable in one line replaces positional arguments.

Reply:
I did not know that and I'm slightly confused. Can you please explain this or point me to an explanation?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
Also, as mentioned in #389, it seems that `local` is not defined by all in POSIX, so that this is another problem.

Reply:
Thanks for the information, I was not aware of this issue!

Reply:
If `local` is not well defined by POSIX, this is a real pity. Just removing it blindly may break some functions, I'm afraid.

Reply:
I raise this question in #389, where almost all other warnings are eliminated. I think, that we might like to use `unset`.